cmake/ProjectWAVM.cmake

```diff
@@ -20,17 +20,19 @@ set(wasm_library ${binary_dir}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}WASM${CMAKE_STA
 set(ir_library ${binary_dir}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}IR${CMAKE_STATIC_LIBRARY_SUFFIX})
 set(logging_library ${binary_dir}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}Logging${CMAKE_STATIC_LIBRARY_SUFFIX})
 set(unwind_library ${binary_dir}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}WAVMUnwind${CMAKE_STATIC_LIBRARY_SUFFIX})
+set(llvmjit_library ${binary_dir}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}LLVMJIT${CMAKE_STATIC_LIBRARY_SUFFIX})
+
+set(other_libraries ${platform_library} ${wasm_library} ${ir_library} ${logging_library} ${unwind_library} ${llvmjit_library})
 
-set(other_libraries ${platform_library} ${wasm_library} ${ir_library} ${logging_library} ${unwind_library})
 
 ExternalProject_Add(wavm
     PREFIX ${prefix}
     DOWNLOAD_NAME wavm-a0baaec170b55cc60cfe6bcc6b36add953a065d8.tar.gz
```

Comment:
We've forgot previously to update the revision here, but should do that now.

Reply:
Not sure what a revision is.

Reply:
The commit hash you are using.

Reply:
Please tell me what to paste and where.

Reply:
@poemm it should be the commit hash of the WAVM commit you are using. that's here, somewhere https://github.com/WAVM/WAVM/commits/master
and probably is `b15e29f2ee361379742957d764aca655ef2a9e48` since you're already using that.

Reply:
like `wavm-b15e29f2ee361379742957d764aca655ef2a9e48.tar.gz`